src/lib/components/Communities/CommunityHeader.js

```diff
@@ -31,15 +31,22 @@ class CommunityHeaderButton extends React.Component {
 class CommunityHeaderInfo extends React.Component {
   render() {
     const { community, imagePlaceholderLink } = this.props;
+    // TODO: Remove checks below once community is resolved in the backend
+    const communityLogo = community?.links?.logo
+      ? community.links.logo
+      : imagePlaceholderLink;
+    const communityTitle = community?.metadata?.title
+      ? community.metadata.title
+      : community;
     return (
       <Container className="community-header">
         <Image
           className="community-logo-header"
-          src={community.links.logo}
+          src={communityLogo}
           fallbackSrc={imagePlaceholderLink}
         />
         <div className="community-header-info">
-          {community.metadata.title}
+          {communityTitle}
```

Comment:
Community is an object right? i think it will error if it is passed instead of the `community.metadata.title` string

Reply:
Depends, it should always be an object but in some cases we can not yet resolve it and we only have the UUID value so we pass this as the `community` value. that's why we check for an object or otherwise we assume it's a string. This is a temporary solution, should be removed as soon as community is resolved in the backend, hopefully in next sprint :)

Reply:
ah ok! That is needed context, thanks.